packages/components/package.json

```diff
@@ -4,19 +4,19 @@
   "description": "React components for Digital Catapult projects.",
   "license": "Apache-2.0",
   "main": "lib/index.js",
-  "module": "src/index.js",
+  "types": "lib/index.d.ts",
```

Comment:
Don't we want to keep "module" and have "types" as an addition? Are these two attributes conflicting or was it removed for another reason?